docker/viame_girder_worker_gpu.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM kitware/viame:girder-worker-gpu-latest
+
+USER root
+
+WORKDIR /home
+
+RUN cd /viame/cmake \
+	&& chmod +x build_server_ubuntu.sh \
+        && sleep 1 \
+ 	&& ./build_server_ubuntu.sh
+
+RUN git clone https://github.com/VIAME/VIAME-Web.git /viame-web \
+	&& mv /viame-web/docker/provision provision \
+	&& mv /viame-web/server viame_girder 
```

Comment:
This is definitely not how this should work.

Reply:
I agree, should this go in `girder.Dockerfile`?

Reply:
I was referring to the fact that this clones master rather than doing a COPY.